tool/speed.cc

```diff
@@ -2628,6 +2642,9 @@ bool Speed(const std::vector<std::string> &args) {
        !SpeedECMUL(selected) ||
        // OpenSSL 1.0 doesn't support Scrypt
        !SpeedScrypt(selected) ||
+#endif
+#if (!defined(OPENSSL_1_0_BENCHMARK) && !defined(BORINGSSL_BENCHMARK) && !defined(OPENSSL_IS_AWSLC)) || AWSLC_API_VERSION >= 24
+        // BoringSSL doesn't support ChaCha, OpenSSL 1.0 doesn't support ChaCha, only AWS-LC after API verison 24
```

Comment:
minor nit: looks like BSSL [supports chacha](https://github.com/google/boringssl/blob/7460d7456433fbaf28ec679fc138fefd6b2120cd/include/openssl/aead.h#L126) through the EVP AEAD API, just not through the EVP Cipher API.

Reply:
Yeah, this benchmark is specifically for EVP_chacha20_poly1305, AWS-LC/BoringSSL wil both test EVP_aead_chacha20_poly1305 down below. I'll update the comment to make it more clear.